dogu.json

```diff
@@ -18,6 +18,11 @@
     "Description": "URL of SCM-Manager Plugin Center",
     "Optional": true
   },
+  {
+    "Name": "scm_plugins_default",
+    "Description": "Comma separated list of plugin names to install on start",
```

Comment:
I think we have to explain that these plugins will be installed on every start, regardless of prior uninstallation.